pootle/templates/includes/formtable.html

```diff
@@ -24,10 +24,8 @@
 		  {% endblocktrans %}
 	      </span>
 	      <span class="pagination-info js-pagination-items-per-page">
-          {% blocktrans count n=formtable.results_per_page trimmed %}
-            Showing {{ n }} result per page
-          {% plural %}
-            Showing {{ n }} results per page
+          {% blocktrans with formtable.results_per_page as n trimmed %}
+          Results per page: {{ n }}
```

Comment:
>  This would take us back to the original which has pluralization issues

to clarify i meant

```
Results per page: {{ formtable.results_per_page }}
```